modules/_labs/breadcrumbs/react/lib/Breadcrumbs/List/hooks.ts

```diff
@@ -15,9 +15,9 @@ export const useCollapse = <E extends HTMLElement>(listEl: React.RefObject<E>, m
 };
 
 const getBreadcrumbLink = (breadcrumb: React.ReactElement) => {
-  return Children.toArray(breadcrumb.props.children).filter(child => {
+  return Children.toArray(breadcrumb.props.children).filter((child: React.ReactElement) => {
     return child.type === BreadcrumbLink;
-  })[0];
+  })[0] as React.ReactElement;
```

Comment:
I was running into an issue with React 17 types. `child` could be a large number of types. The safer fix would be to use a guard type to determine `BreadcrumbLink` children, but it was easier to restrict and cast.